subsys/bluetooth/services/Kconfig.bas

```diff
@@ -11,6 +11,6 @@ if BT_BAS
 
 module = BT_BAS
 module-str = BAS
-source "${ZEPHYR_BASE}/subsys/logging/Kconfig.template.log_config"
+source "subsys/logging/Kconfig.template.log_config"
```

Comment:
definitely not an expert but shouldn't this be `rsource`?

Reply:
`rsource` would mean that the `subsys/logging/Kconfig.template.log_config` is expected below `subsys/bluetooth/services/`. For example here: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/audio/Kconfig#L28 `rsource` mean that the included Kconfig is in path relative to the one where currently processed  Kconfig is placed.